tests/functional/ecr/test_get_login_password.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Let's update this copyright to 2020 but otherwise I think this looks fine.

Reply:
updated